tools/pythonpkg/tests/fast/test_multithread.py

```diff
@@ -415,15 +415,13 @@ def test_fetchdfchunk(self, duckdb_cursor, pandas):
 
     @pytest.mark.parametrize('pandas', [NumpyPandas(), ArrowPandas()])
     def test_fetcharrow(self, duckdb_cursor, pandas):
-        if not can_run:
-            return
+        pytest.importorskip('pyarrow')
```

Comment:
Using importorskip here seems a lot more idiomatic than the `can_run` approach. Hopefully that's fine :)

Reply:
I am replacing them whenever I touch files with this in them, thanks for this 👍